src/main/groovy/com/okta/test/mock/tests/CodeFlowLocalValidationIT.groovy

```diff
@@ -84,7 +84,7 @@ class CodeFlowLocalValidationIT extends ApplicationTestRunner {
     void respondWithCode() {
         ExtractableResponse response = redirectToRemoteLogin()
         String redirectUrl = response.header("Location")
-        String state = redirectUrl.substring(redirectUrl.lastIndexOf('=')+1)
+        String state = TestUtils.parseQuery(new URL(redirectUrl).query).get("state").get(0)
```

Comment:
Curious if we could enter a state where `new URL(redirectUrl).query` could fail. Since we're wrapping a bit of conversion logic in a single line, would it be better to assert `null` values?

Reply:
`redirectToRemoteLogin()` validates the redirectUrl will be in the correct format otherwise an test assert error will be thrown

Reply:
In this case the test would have failed in `redirectToRemoteLogin()` if `state` wasn't at least 6 characters long.  The assert message that would be thrown is a bit verbose, possibly too much, but I figured that would be better than not providing enough context.